matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/failure/Extensions.kt

```diff
@@ -24,6 +24,10 @@ import org.matrix.android.sdk.internal.di.MoshiProvider
 import java.io.IOException
 import javax.net.ssl.HttpsURLConnection
 
+fun Failure.ServerError.isTokenUnknownError() =
```

Comment:
Can we keep the format `fun Throwable.isTokenUnknownError()` and check the type as per the other extensions in this file?